channels-and-providers/sms/46elks.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: '46elks'
+description: 'Learn how to use the 46elks provider to send sms notifications using Novu'
+---
+
+[46elks](https://46elks.com/) API let's you intigrate SMS, MMS and phone calls into your applications with just a few lines of code.
```

Comment:
Please rephrase to:


You can use the [46elks](https://46elks.com/) provider to send SMS messages to your customers using the Novu platform with a single API to create multi-channel experiences.